addons/payment_authorize/tests/test_authorize.py

```diff
@@ -33,13 +33,26 @@ def setUp(self):
 class AuthorizeForm(AuthorizeCommon):
 
     def _authorize_generate_hashing(self, values):
+        def is_signature_key(key):
```

Comment:
why all this code is duplicated in test ?

imo don't need to test old behavior

Reply:
> why all this code is duplicated in test ?

Well, the code from `_authorize_generate_hashing` is already duplicated in tests.

> imo don't need to test old behavior

I prefer to not leave the old hashing method in tests since it would raise an error when running tests locally with the new signature key.
But since data still contain 'dummy' we could leave tests as they are but they won't reflect reality IMHO

The best way would be to just extract the `_authorize_generate_hashing` so it is not a class method anymore and so it could be imported (it does not need self anyway).
and the in tests
```python
from odoo.addons.payment_authorize.models.payment import _authorize_generate_hashing
```
But in 10.0.. Not sure it is a good idea.

Reply:
As discussed, it will remain a class method and 
```python
self.env['payment.acquirer']._authorize_generate_hashing(form_values)
```
will be enough on tests